docs/getting_started.md

```diff
@@ -355,5 +355,8 @@ Please note: We strongly discourage connecting FLUFFI to an untrusted network (s
 
 ## A Note on Operational Security
 
-FLUFFI was not meant to be used in hostile environments, which is why FLUFFI's internal protocol is not hardened against attacks. It is therefore recommended to operate FLUFFI in an isolated environment, to which you can restrict access, and only allow the master to be connected to the internet via NAT.
-Furthermore, it is strongly recommended to change the Polemarch user credentials, which are initially set to be `admin:admin`.
+FLUFFI was not meant to be used in hostile environments, which is why FLUFFI's internal protocol is not hardened against attacks.
+It is therefore recommended to operate FLUFFI in an isolated environment, to which you can restrict access, and only consider allowing the master to connect to the internet via NAT.
+Please note that there are currently several outdated dependencies, including at least one known vulnerability in the django version used in the Polemarch subcomponent.
+Since it is very fragile, changing any dependency version in this component bears the risk of loosing its functionality.
```

Comment:
"Since this component is very fragile, changing any dependency version in it..."
This way it is clearer what "it" is referring to :)